x/twap/strategy_test.go

```diff
@@ -166,6 +162,34 @@ func (s *TestSuite) TestComputeArithmeticStrategyTwap() {
 // Contrary to computeTwap function (logic.go) that handles the cases with zero delta correctly,
 // this function should panic in case of zero delta.
 func (s *TestSuite) TestComputeGeometricStrategyTwap() {
+	var (
+		errTolerance = osmomath.ErrTolerance{
+			MultiplicativeTolerance: sdk.SmallestDec(),
+			RoundingDir:             osmomath.RoundDown,
+		}
+
+		// Compute accumulator difference for the underflow test case by
+		// taking log base 2 of the min spot price
+		minSpotPriceLogBase2 = twap.TwapLog(gammtypes.MinSpotPrice)
+
+		// Compute accumulator difference for the overflow test case by
+		// taking log base 2 of the max spot price
+		maxSpotPriceLogBase2 = twap.TwapLog(gammtypes.MaxSpotPrice)
+
+		oneHundredYearsInHours        int64 = 100 * 365 * 24
+		oneHundredYears                     = OneSec.MulInt64(60 * 60 * oneHundredYearsInHours)
+		oneHundredYearsMin1MsDuration       = time.Duration(oneHundredYearsInHours)*time.Hour - time.Millisecond
+
+		// Subtract 1ms from 100 years to assume that we interpolate.
+		oneHundredYearsMin1Ms = oneHundredYears.Sub(oneDec)
```

Comment:
Is oneDec in this case 1ms or a second?

Reply:
This should be in milliseconds. That is one dec = 1ms